packages/reports/tests/integration/components/report-actions/export-test.js

```diff
@@ -59,7 +59,7 @@ module('Integration | Component | report actions - export', function(hooks) {
           type: 'timeDimension',
           source: 'bardOne',
           field: 'network.dateTime',
-          parameters: { grain: 'day' },
+          parameters: { grain: 'all' },
```

Comment:
Did this need to change?

Reply:
No it didnt need to change I double checked I think I ran into an error when I first started fixing the test, but when I just ran the tests as 'day' it went fine. Do you want me to change it back?

Reply:
Yea change it back to `'day'` 👍